styles.css

```diff
@@ -111,3 +111,105 @@
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width: 1560px) {
+  .wrapper {
+    width: 69%;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6306727)